site/css/typography/_fonts.css

```diff
@@ -96,6 +96,11 @@
   line-height: 1.2em;
 }
 
+.fontM3 {
+  font-size: 24px;
```

Comment:
I think we shouldn't add new typography styles as everything we use should be taken from: http://brand.red-badger.com/d/WEl30uS4YzKw/red-badger-styleguide-style-guide#/elements/typography